charts/consul/templates/crd-gateways.yaml

```diff
@@ -7,14 +7,15 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     api-approved.kubernetes.io: https://github.com/kubernetes-sigs/gateway-api/pull/1538
-    gateway.networking.k8s.io/bundle-version: v0.6.2
-    gateway.networking.k8s.io/channel: experimental
   labels:
     app: {{ template "consul.name" . }}
     chart: {{ template "consul.chart" . }}
     heritage: {{ .Release.Service }}
     release: {{ .Release.Name }}
     component: crd
+    gateway.networking.k8s.io/bundle-version: v0.6.2
```

Comment:
Did you mean to touch all of these templates?

Reply:
I had to when I regen'd the CRDs

Reply:
I think the person who did this before didn't use the `0.8.0` version that is standard for the repo.